src/main/java/listeners/TestListener.java

```diff
@@ -33,77 +33,75 @@
 
     @Override
     public void onTestStart(ITestResult result) {
-        testName = result.getName();
-        logger.info("Test [" + testName + "] has started");
+        logger.info(String.format("Test [%s] has started", result.getName()));
```

Comment:
In case of logging you can use logger.info("Test {} has started", result.getName());

Reply:
Update it everywhere